cargo-bolero/Cargo.toml

```
@@ -14,13 +14,15 @@ readme = "README.md"
 default = ["afl", "libfuzzer"]
 afl = ["bolero-afl"]
 honggfuzz = ["bolero-honggfuzz"]
+rmc = ["bolero-rmc"]
```

Since we're not linking to anything we can just have a feature without the optional dependency

```suggestion
rmc = []
```